modules/cassandra/src/main/java/org/testcontainers/containers/delegate/CassandraDatabaseDelegate.java

```diff
@@ -0,0 +1,58 @@
+package org.testcontainers.containers.delegate;
+
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.exceptions.DriverException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.containers.CassandraContainer;
+import org.testcontainers.containers.ContainerState;
+import org.testcontainers.delegate.AbstractDatabaseDelegate;
+import org.testcontainers.exception.ConnectionCreationException;
+import org.testcontainers.ext.ScriptUtils.ScriptStatementFailedException;
+
+/**
+ * Cassandra database delegate
+ *
+ * @author Eugeny Karpov
+ */
+@Slf4j
+@RequiredArgsConstructor
+public class CassandraDatabaseDelegate extends AbstractDatabaseDelegate<Session> {
+
+    private final ContainerState container;
+
+    @Override
+    protected Session createNewConnection() {
+        try {
+            return CassandraContainer.getCluster(container)
```

Comment:
Same question as above, why not calling `getCluster()` on the container?

Reply:
see the answer :)